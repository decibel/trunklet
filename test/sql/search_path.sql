\set ECHO none
\i test/pgxntool/setup.sql

SELECT plan(
    0

    +1 -- verify search_path
);

CREATE TEMP TABLE before AS
  SELECT current_setting('search_path')
;

SHOW search_path;

CREATE EXTENSION trunklet;-- CASCADE;

CREATE TEMP TABLE after AS
  SELECT current_setting('search_path')
;

-- Forcibly reset it so we know tap will work
SET search_path=public,tap;

SELECT is(
  (SELECT current_setting FROM after)
  , (SELECT current_setting FROM before)
  , 'verify search_path has not changed'
);

\i test/pgxntool/finish.sql

-- vi: expandtab ts=2 sw=2
