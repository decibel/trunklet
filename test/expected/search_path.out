\set ECHO none
1..1
tap, public
ok 1 - verify search_path has not changed
# TRANSACTION INTENTIONALLY LEFT OPEN!
